aws/monolith-terraform/xebialabs/xld-service.yaml.tmpl

```diff
@@ -0,0 +1,34 @@
+{{$app := .AppName | kebabcase}}
+{{$region := "{{AWSRegion}}"}}
```

Comment:
missing `.` before variables

Reply:
this is a placeholder that should be replaced by the value defined in the dictionary. I defined like this to centralize the value, since it is defined in multiple CIs